biztown/src/components/DepositForm.js

```diff
@@ -1,204 +1,230 @@
-                            <div className="deposit-form-small-text center-align">Deposits may not be available for immediate withdrawal.</div>
-                            <div
-                                className="table-border-bottom center-align"
-                                style={{fontFamily: "cursive", marginTop:"15px", marginRight: "15px"}}
-                            >
-                                Mark Hill
-                            </div>
-                            <div
-                                className="deposit-form-small-text center-align"
-                                style={{padding: "0 20px"}}
-                            >
-                                <i>Signature required for cash received</i>
-                            </div>
-                        </div>
-                        <div style={{marginTop: "30px"}}>
-                            <DepositFillableTable
-                                handleChange={this.props.handleChange}
-                                rowData={this.props.data}
-                            />
-                            <div className="flex-container" style={{marginLeft: "10px", marginTop: "15px"}}>
-                                <span>Acct. #</span>
-                                <span
-                                    className="flex-item-grow table-border-bottom"
-                                    style={{textAlign: "center", marginLeft: "10px"}}
-                                >
-                                    132
-                                </span>
-                            </div>
-                        </div>
-                    </div>
-                </div>
+  render() {
+    return (
+      <div
+        className="deposit-form flex-container"
+        style={{ overflowX: "scroll" }}
+      >
+        <div
+          className="deposit-form flex-container"
+          style={{
+            height: "240px",
+            width: "700px",
+            border: "1px solid black",
+            flex: "0 0 auto",
+            flexDirection: "column",
+            padding: "15px",
+          }}
+        >
+          <div className="flex-container">
+            <div>
+              <div
+                className="center-align"
+                style={{ marginBottom: "30px" }}
+              >
+                <b>DEPOSIT TICKET</b>
+              </div>
+              <div className="flex-container">
+                <span>
+                  <b>Name</b>
+                </span>
+                <span
+                  className="flex-item-grow table-border-bottom"
+                  style={{
+                    textAlign: "center",
+                    marginRight: "15px",
+                    marginLeft: "10px",
+                    width: "275px",
+                  }}
+                >
+                  Mark Hill
+                </span>
+              </div>
+              <img
+                src={bizTownBank}
```

Comment:
Very much my b, but chrome doesn't appear to be happy with this.

<img width="371" alt="Screen Shot 2021-03-17 at 2 14 01 AM" src="https://user-images.githubusercontent.com/28936879/111423305-7769c880-86c6-11eb-97d9-05e38a99cbc4.png">

I think what should do the trick is requiring this as a variable instead of importing it. i.e. something along the lines of `bizTownBank = require("../images/bizTownLogo.png")`.

What I think is happening is when you import a file, it is loaded into the browsers assets folder (or whatever the browser is set up to do) and Chrome is not finding the image where it expects it. `require` helps a bit with that. I like to think about it as relative path (import) versus absolute path (require). This is overly simplified and is a small part of the required functionality, but if you are interested here is a link to [the nodejs documentation of this](https://nodejs.org/api/modules.html#modules_require_id) and a [stackoverflow link](https://stackoverflow.com/questions/9901082/what-is-this-javascript-require) of people super hashing it out.

Reply:
Hmm, odd. It works on my chrome browser. I couldn't get it to work with require, so I've added this to the backlog.